sentry_telegram/plugin.py

```diff
@@ -124,6 +138,12 @@ def send_message(self, url, payload, receiver):
         )
         self.logger.debug('Response code: %s, content: %s' % (response.status_code, response.content))
 
+    def send_message_through_proxy(self, url, payload, receiver, proxy):
+        payload['chat_id'] = receiver
+        self.logger.debug('Sending message to %s . Using proxy' % receiver)
+        response = requests.post(url, data=payload, proxies=dict(http=proxy, https=proxy))
```

Comment:
I would prefer to refrain from the direct use of `requests`. Sentry has internal, unified utils to interact with network.
Please, look for [safe_urlopen()](https://github.com/getsentry/sentry/blob/master/src/sentry/http.py#L179) built-in function. It supports `params` to pass additional options, like proxy-settings.

Reply:
Hey, there is already one https://github.com/getsentry/sentry/pull/6784